pages/activities.jsx

```diff
@@ -46,13 +49,17 @@ var ActivityKit = React.createClass({
     // Displays the last author with an "and" and an oxford comma.
     var _this = this;
     var hasDash = this.props.developedBy.indexOf("-") > -1;
-    var developedByArray = this.props.developedBy.replace("-", ",").split(",");
+    var hasInPartnershipWith = this.props.developedBy.indexOf("in partnership with") > -1;
+    var developedByArray = this.props.developedBy.replace("-", ",").replace("in partnership with", ",").split(",");
     var developedByElements = developedByArray.map(function (author, key) {
       author = author.trim();
       var separator = ", ";
       if (key === 0 && hasDash) {
-        separator = " - "
-      } else if (key === developedByArray.length-1) {
+        separator = " - ";
+      } else if (key === 0 && hasInPartnershipWith) {
+        separator = " in partnership with ";
+      }
```

Comment:
I didn't wanna have a nested if-else statements here (for `key===0` cases) so I kept this separate